packages/js/components/src/utils.tsx

```diff
@@ -64,7 +64,7 @@ function createOrderedChildren< T = Fill.Props, S = Record< string, unknown > >(
 	order: number,
 	props: T,
 	injectProps?: S
-) {
+): React.ReactSVGElement {
```

Comment:
I think this is unintended? `React.ReactElement` seems to be the better suited type
```suggestion
): React.ReactElement {
```

Reply:
I went with what VS Code told me that the function would be returning 😄 Thanks for the tip here, I'll make the change!